ps/jenkins/package-testing-ps-5.7.groovy

```diff
@@ -0,0 +1,276 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+List all_nodes = [
+    "min-stretch-x64",
+    "min-buster-x64",
+    "min-centos-6-x64",
+    "min-centos-7-x64",
+    "min-centos-8-x64",
+    "min-xenial-x64",
+    "min-bionic-x64",
+    "min-focal-x64",
+    "micro-amazon",
+]
+
+Map product_nodes = [
+    ps56: [
+        "min-stretch-x64",
+        "min-centos-6-x64",
+        "min-centos-7-x64",
+        "min-xenial-x64",
+        "min-bionic-x64",
+        "micro-amazon",
+    ],
+    ps57: all_nodes,
+    ps80: all_nodes,
+    client_test: all_nodes,
+]
+
+product_to_test = params.product_to_test
+
+List nodes_to_test = []
+if (params.node_to_test == "all") {
+    nodes_to_test = all_nodes
+} else {
+    nodes_to_test = [params.node_to_test]
+}
+
+void runNodeBuild(String node_to_test) {
+    build(
+        job: 'package-testing-ps-build',
+        parameters: [
+            string(name: "product_to_test", value: product_to_test),
+            string(name: "install_repo", value: params.install_repo),
+            string(name: "client_to_test", value: params.client_to_test),
+            string(name: "node_to_test", value: node_to_test),
+            string(name: "action_to_test", value: params.action_to_test)
+        ],
+        propagate: true,
+        wait: true
+    )
+}
+
+pipeline {
+    agent none
+
+    parameters {
+        choice(
+            name: "product_to_test",
+            choices: ["ps57", "ps56", "client_test"],
```

Comment:
well as I can see this job is dedicated for 5.7 version. Why do we add 5.6?

Reply:
After discussing this with the QA team, we'll remove support for 5.6 from these Jenkins jobs. It's unlikely we'll need to run 5.6 tests again, which is why they didn't get their own Jenkins job, but for that same reason it's okay to run them manually if we ever need to.